lib/ansible/modules/cloud/openstack/os_quota.py

```diff
@@ -287,6 +287,7 @@
 '''
 
 import sys
+from keystoneauth1 import exceptions
```

Comment:
shouldn't this be in a `try` to capture import errors and give a useful message to user?